content/20.treatments.md

```diff
@@ -224,17 +224,23 @@ Summarize the state of the antiviral approach.
-A case study of three COVID-19 patients reported presence of antiphospholipid antibodies in all three patients [@doi:10.1056/NEJMc2007575].
-Antiphospholipid antibodies are central to the diagnosis of the antiphospholipid syndrome. 
-HCQ has been reported to be effective in treating Antiphospholipid Syndrome [@doi:10.1182/blood.V128.22.5023.5023; @doi:10.1016/j.autrev.2014.01.053; @doi:10.1182/asheducation-2016.1.714].
-
-Thus, there is excitement about HCQ as a potential therapeutic for SARS-CoV-2 and COVID-19.
-This section will discuss current efforts on assessing the potential impact of HCQ in treating COVID-19.
+CQ and HCQ are lysosomotropic agents, meaning they are weak bases that can pass through the plasma membrane.
+Both drugs increase cellular pH by accummulating in their protonated form inside lysosomes [@doi:10.1186/1743-422X-8-163; @doi:10.1111/j.1529-8019.2007.00131.x].
+This shift in pH inhibits the breakdown of proteins and peptides by the lysosomes during the process of proteolysis [@doi:10.1111/j.1529-8019.2007.00131.x].
+A number of mechanisms have been proposed through which these drugs could influence the immune response to pathogen challenge.
+For example, CQ/HCQ can interfere with digestion of antigens within the lysosome and inhibit CD4 T-cell stimulation while promoting the stimulation of CD8 T-cells.
+CQ/HCQ can also decrease the production of certain key cytokines involved in the immune response including IL-6 and inhibit the stimulation of toll-like receptors (TLR) and TLR signaling [@doi:10.1111/j.1529-8019.2007.00131.x].
+The drugs also have anti-inflammatory and photoprotective effects and may also affect rates of cell death, blood clotting, glucose tolerance, and cholesterol [@doi:10.1111/j.1529-8019.2007.00131.x].
+
+Interest in CQ and HCQ for treating COVID-19 was catalyzed by a mechanism observed in _in vitro_ studies of both SARS-CoV and SARS-CoV-2.
+In one study, CQ inhibited viral entry of SARS-CoV through the elevation of endosomal pH and the terminal glycosylation of angiotensin-converting enzyme 2 (ACE2), which is the cellular entry receptor [@doi:10.1186/1743-422X-2-69].
+Increased pH within the cell, as discussed above, inhibits proteolysis, and terminal glycosylation of ACE2 is thought to interfere with virus-receptor binding.
+_In vitro_ studies of SARS-CoV-2 infection show both HCQ and CQ to be effective in inhibiting viral replication, with HCQ being more potent [@doi:10.1093/cid/ciaa237].
+Additionally, an early case study of three COVID-19 patients reported the presence of antiphospholipid antibodies in all three patients [@doi:10.1056/NEJMc2007575].
+Antiphospholipid antibodies are central to the diagnosis of the antiphospholipid syndrome, and HCQ has been reported to be effective in treating Antiphospholipid Syndrome [@doi:10.1182/blood.V128.22.5023.5023; @doi:10.1016/j.autrev.2014.01.053; @doi:10.1182/asheducation-2016.1.714].
```

Comment:
```suggestion
Antiphospholipid antibodies are central to the diagnosis of the antiphospholipid syndrome, and HCQ has a history of use in treating Antiphospholipid Syndrome [@doi:10.1182/blood.V128.22.5023.5023; @doi:10.1016/j.autrev.2014.01.053; @doi:10.1182/asheducation-2016.1.714].
```
I didn't see a randomized, controlled trial here so I softened the language a bit.